### PR TITLE
SG-26481_sg Set Python 3 for Desktop

### PR DESCRIPTION
**Pull request by francoperroneautodesk**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/4765de5ae214df84f673dcb8c50c22b713c3f803/index.html)

**Comment by mathurf** [comment]
This image will also need to be updated. Value should be 2 now
image

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/cfa75d44f85e581df24557db8c3d63ac6fb3d910/index.html)

**Comment by francoperroneautodesk** [comment]
> @000paradox000 can you help Franco with this documentation page changes. Also look at the Jira ticket, we might want to add something about the fact that Python 2.7 will be deprecated on Nov 1st 2022 and adding a link to Warren's release notes about it.
> 
> Thanks!

Ariell will solve this in a new ticket

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/dc6b19133ebef2cbc3189273bb880aa92645eff8/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/c5c7f167d9fcff28d33f17eee0f9da751276863c/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/d898f64bb22d8530f9b20bd79cb16d5b09dcb690/index.html)

File: docs/en/guides/pipeline-integrations/administration/setting-python-3-desktop.md [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
 
 ## Windows
 
-### Manually Set the `SHOTGUN_PYTHON_VERSION` environment to 3 on Windows
+### Manually Set the `SHOTGUN_PYTHON_VERSION` environment to 2 on Windows
 
-- On the Windows taskbar, right-click the Windows icon and select **System**, navigate through the **Control Panel/System and Security/System**. 
+- On the Windows taskbar, right-click the Windows icon and select **System**, navigate through the **Control Panel/System and Security/System**. 
 
 ![](images/setting-python-3-desktop/01-setting-python-3-desktop.png)
```
Comment (mathurf) [inline docs/en/guides/pipeline-integrations/administration/setting-python-3-desktop.md:23]: Line 36 value should be 2
- For the **Variable name**, add `SHOTGUN_PYTHON_VERSION`, and set the **Variable value** to `3`.

File: docs/en/guides/pipeline-integrations/administration/setting-python-3-desktop.md [hunk 2]
```diff
@@ -42,15 +44,15 @@ lang: en
```
Comment (mathurf) [inline docs/en/guides/pipeline-integrations/administration/setting-python-3-desktop.md:44]: Line 40 it should mention Python 2
- Restart the {% include product %} Desktop application. Now, you should see that the Python version has been updated to run Python 3.

**Review by mathurf — CHANGES_REQUESTED** [review]

**Review by mathurf — APPROVED** [review]
LGTM!

**Pull request merged**